roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -34,7 +34,7 @@ def run(self, tmp, task_vars):
             }
 
         # attempt to get the docker info hash from the API
-        info = self.execute_module("docker_info", {}, task_vars)
+        info = self.execute_module("docker_info", {}, task_vars=task_vars)
```

Comment:
We can set variables at the time the checks are instantiated (which happens inside running our action plugin, so vars should be accurate). We only ever run them once per instantiation. I had a branch where I coded all this out a few months ago but never worked up a proper PR.